src/main.cpp

```diff
@@ -39,10 +40,40 @@ main (int argc, char *const argv[])
     std::vector<function> functions = std::move (ctx.get_function_list ());
 
     for (const auto &f : functions)
-	{
-	    std::cout << f;
-	}
+    {
+        std::cout << f;
+    }
 
+    #ifdef PARSING_VISUALIZATION_ENABLED
+        std::string stringified;
+        for (const auto &f : functions)
+        {
+//            std::cout << "---------------------------------\n";
```

Comment:
Avoid committing large chunks of comments.

Reply:
Sorry, my fault. Will be fixed in next commit